Index.html

```diff
@@ -0,0 +1,177 @@
+    </h2>
+    <!-- boxes of grid -->
+    <div class="main-program__grid-container">
+    <div class="main-program__box box1">
+      <img src="img/Icon-1.svg" alt="" class="main-program__box__logo">
+      <h3 class="main-program__box__h3">
+        Lecture
+      </h3>
+      <p class="main-program__box__p">
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Minus debitis iure reiciendis ipsa, veniam laudantium.
+      </p>
+    </div>
+
+    <!-- boxes of grid -->
+    <div class="main-program__box box2">
+      <img src="img/Icon-2.svg" alt="" class="main-program__box__logo">
+      <h3 class="main-program__box__h3">
+        Ignite
+      </h3>
+      <p class="main-program__box__p">
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Minus debitis iure reiciendis ipsa, veniam laudantium.
+      </p>
+    </div>
+
+    <!-- boxes of grid -->
+    <div class="main-program__box box3">
+      <img src="img/Icon-3.svg" alt="" class="main-program__box__logo">
+      <h3 class="main-program__box__h3">
+        Forum
+      </h3>
+      <p class="main-program__box__p">
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Minus debitis iure reiciendis ipsa, veniam laudantium.
+      </p>
+    </div>
+
+    <!-- boxes of grid -->
+    <div class="main-program__box box4">
+      <img src="img/Icon-1.svg" alt="" class="main-program__box__logo">
+      <h3 class="main-program__box__h3">
+        Workshop
+      </h3>
+      <p class="main-program__box__p">
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Minus debitis iure reiciendis ipsa, veniam laudantium.
+      </p>
+    </div>
+    </div>
+    <div class="main-program__btn">
+      Join Our Program
+    </div>
+    <div>
+      <a href="#" class="main-program__btn__desk">SEE THE WHOLE PROGRAM</a>
+    </div>
+  </section>
+  <section class="feature">
+    <h2 class="feature__h2">
+      Featured Speakers
+      <span class="red-bar"></span>
+    </h2>
+    <div class="feature__grid-container"></div>
+  </section>
+  <section class="more">
+    <div class="more__div">
+    <p class="more__p">
+      MORE
+    </p>
+    <span class="material-symbols-outlined more__arrow">
+      expand_more
+    </span>
+  </div>
+  </section>
+  <section class="partner partner__desk">
+    <h2 class="partner__h2">
+      Partners
+      <span class="red-bar"></span>
+    </h2>
+    <div class="partner__logo">
+      <img src="img/CodeClub.svg" alt="">
+      <img src="img/NOVER.svg" alt="">
+      <img src="img/mezilla.svg" alt="">
+      <img src="img/CodeClub.svg" alt="">
+      <img src="img/mezilla.svg" alt="">
+    </div>
+    </section>
+    <footer>
+      <div class="footer-flex footer-flex__desk">
+      <div class="footer__logo">
+        <img src="img/microverse-logo.png" alt="" class="footer__logo__img">
+      </div>
+      <div class="footer__p">
+        2023 M1 Capstone project.<br>No Rights Reserved.
+      </div>
+     </div>
```

Comment:
- [ ] The footer of the desktop index page has different colors from the template. The template use a white background color with dark text and logo. Kindly change the colors to match the template.
 - [OPTIONAL] It would be nice to position the logo and text a little bit to the center too. 
 - Your design:
 - 
<img width="1115" alt="image" src="https://user-images.githubusercontent.com/61919617/217387344-1c2a1767-0cc9-4622-a912-353f9b29ea92.png">

 - Template:
 - 
<img width="1027" alt="image" src="https://user-images.githubusercontent.com/61919617/217387416-4c3614bd-767b-431e-b1d9-bc842f80c2c9.png">